Graphics/Core/include/NovelRT/Graphics/GraphicsPipelineInput.hpp

```diff
@@ -1,12 +1,11 @@
+#pragma once
+
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
 // for more information.
 
-#ifndef NOVELRT_GRAPHICS_GRAPHICSPIPELINEINPUT_H
-#define NOVELRT_GRAPHICS_GRAPHICSPIPELINEINPUT_H
-
-#ifndef NOVELRT_GRAPHICS_H
-#error NovelRT does not support including types explicitly by default. Please include Graphics.h instead for the Graphics namespace subset.
-#endif
+#include <vector>
+#include <NovelRT/Utilities/Misc.h>
+#include <NovelRT/Graphics/GraphicsPipelineInputElement.hpp>
```

Comment:
Should these get flipped? I'm not sure if we have a well-defined pattern, but other files previously did:
```cpp
#include <NovelRT/blah/blah.h>
#include <vector>
```
(As an example, we have been putting NovelRT references before stdlib and others)

Reply:
uhhhh clang-format should have re-arranged these at some point? I can try to verify if you want.

Reply:
If CI isn't really complaining, then I guess it's okay

Reply:
Looks like CI clang-format didn't catch this one, my local clang-format did. Thanks!